src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -347,6 +351,11 @@ export class SkyAutocompleteComponent
   }
 
   private sendDropdownMessage(type: SkyDropdownMessageType): void {
+    if (type === SkyDropdownMessageType.Open) {
+      this._dropdownIsOpen = true;
```

Comment:
To avoid side effects, could we make the `isDropdownOpen` assignment before calling `sendDropdownMessage`? We're only calling this method twice (when it concerns the open/closed status of the dropdown).
```
this.isDropdownOpen = false;
this.sendDropdownMessage(SkyDropdownMessageType.Close);
```

Reply:
Sure thing, updated.